### PR TITLE
17.0 technical onboarding abdu

### PR DESCRIPTION
**Pull request by mohammadhossam**

### CODE REVIEW

File: estate/models/estate_property.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    garden = fields.Boolean()
+    garden_area = fields.Integer()
+    garden_orientation = fields.Selection(
+        string='Type',
+        selection=[('north', 'North'), ('south', 'South'),
+                   ('east', 'East'), ('west', 'West')]
+    )
+    active = fields.Boolean(default=True)
+    state = fields.Selection(required=True, selection=[('new', 'New'), ('offer_received', 'Offer Received'), (
+        'offer_accepted', 'Offer Accepted'), ('sold', 'Sold'), ('canceled', 'Canceled')], default='new', copy=False)
```
Comment (hupo-odoo) [inline estate/models/estate_property.py:29]:
```suggestion
    state = fields.Selection(
        selection=[
            ('new', 'New'), 
            ('offer_received', 'Offer Received'), 
            ('offer_accepted', 'Offer Accepted'), 
            ('sold', 'Sold'), 
            ('canceled', 'Canceled')
        ], 
        default='new', 
        required=True, 
        copy=False,
    )
```
Selection fields are more readable like this. You can also do something like this for the other ones

File: estate/security/ir.model.access.csv [hunk 2]
```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+estate.access_estate_property,access_estate_property,estate.model_estate_property,base.group_user,1,1,1,1
```
Comment (hupo-odoo) [inline estate/security/ir.model.access.csv:2]:
```suggestion
estate.access_estate_property,access.estate.property,estate.model_estate_property,base.group_user,1,1,1,1
```
the name of a model should be separated with dots, not underscores.

File: estate/models/estate_property.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
+from email.policy import default
+from odoo import models, fields
+
+
+class EstateProperty(models.Model):
+    _name = "estate_property"
```
Comment (hupo-odoo) [inline estate/models/estate_property.py:6]: The name of a model should always be separated with dots, not underscore
```suggestion
    _name = "estate.property"
```

File: estate/models/estate_property.py [hunk 4]
```diff
@@ -0,0 +1,29 @@
+from email.policy import default
```
Comment (hupo-odoo) [inline estate/models/estate_property.py:1]: you don't need this import
```suggestion
```

**Comment by mohammadhossam** [comment]
@hupo-odoo Thank you for the suggestions. They should be resolved by now.

File: estate/views/estate_property_views.xml [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+                        <h1 class="mb16">
+                            <field name="name" placeholder="My new house" class="mb16" />
+                        </h1>
+                    </div>
+                    <field name="tag_ids" widget="many2many_tags" />
+                    <group>
+                        <group>
+                            <field name="state" />
+                            <field name="property_type_id" />
+                            <field name="postcode"></field>
```
Comment (hupo-odoo) [inline estate/views/estate_property_views.xml:47]:
```suggestion
                            <field name="postcode"/>
```
No need for closing tag if nothing inside

File: .gitignore [hunk 6]
```diff
@@ -127,3 +127,6 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+# VS Code settings
+.vscode/
```
Comment (hupo-odoo) [inline .gitignore:132]: This file shouldn't be in the PR. I guess you can configure your IDE to ignore it

**Review by hupo-odoo — COMMENTED** [review]
Very good work :+1: I've made a few suggestions for improvement. Keep up :partying_face:

**Pull request closed without merging**